modules/menu/menu.c

```diff
@@ -551,6 +551,9 @@ static void menu_play_closed(struct call *call)
 
 		menu_play(call, key, fb, 1, DEVICE_ALERT);
 	}
+	else {
+		menu_play(call, "hangup_aufile", "none", 0, DEVICE_PLAYER);
```

Comment:
Are you/we sure that it should be played on the same device where the speech can be heard?

- DEVICE_PLAYER: speech, call-waiting, ringback (for outgoing calls)
- DEVICE_ALERT: incoming ringtone, error tones

E.g. when a headset is used for the call.

I would also suggest to play it on `DEVICE_PLAYER`.

This PR looks good now for me.